scripts/plugins/firewall-port

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+#
+# Copyright (c) Citrix Systems 2017. All rights reserved.
+
+set -e
+
+# Using this script to open/clear port. 
+
+
+OP="$1"
+PORT="$2"
+CHAIN="RH-Firewall-2-INPUT"
+RULE="-p tcp -m conntrack --ctstate NEW -m tcp --dport $PORT -j ACCEPT"
+
+case "${OP}" in
+    open)
+        if iptables -S | grep $CHAIN | grep -e "${RULE}" > /dev/null
+        then
+            echo $"port $2 is opened already"
+        else
+            if iptables -S | grep $CHAIN | grep -e "${CAHIN}" > /dev/null
+            then
+                echo $"$CHAIN already exists"
+            else
+                iptables -N $CHAIN
```

Comment:
This creates a new chain, but will it ever be used?
You need some change elsewhere as well.